### ISSUE TITLE
Funds show only two first currency

### ISSUE THREAD

**Issue by lemial** [peatio/peatio#443]
Hi!

The menu displays only two Funds currency. Prompt where to display all correct me added to currency.yml.

**Comment by alberthendriks** [comment]
+1

**Comment by alberthendriks** [comment]
In my case, the problem was suddenly solved. The last thing I did before that is: make a deposit in my added fiat currency and confirm the deposit.

**Comment by charnasol** [comment]
Question: did you have any issues getting the main trading board to quote bitcoin pricing/stream the pricing and what files seemed to make sure it worked. That's problem I'm having and hoping for help if possible

**Comment by luiskahn718** [comment]
so how to solve this issue? I have same issue on funds page

**Comment by lemial** [comment]
Yes, solve, add currency number to DB table contains user balance!

**Comment by luiskahn718** [comment]
what does that mean? Can you tell me that step by step? I am new to peatio:)
